step-capstone/src/components/AddFormPopover.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import {Popover} from '@material-ui/core';
+import AddForm from './AddForm'
+
+export default function AddFormPopver(props) {
```

Comment:
Curious: How well does this popover perform on a variety of window and screen sizes?

Reply:
it's position and sizing will properly adjust itself if the window size changes. If it becomes too small it will become scrollable.